addons/necropost-highlighter/addon.json

```diff
@@ -0,0 +1,143 @@
+{
+  "name": "Necropost highlighter",
+  "description": "Spot old forum topics brought back to life, before opening the message. Topics started much earlier than others listed on the same page are highlighted (in selected forums). If you've wasted time reading zombie messages or a help/question/request that was actually asked years ago, this is for you!",
```

Comment:
```suggestion
  "description": "In the selected forums, highlights topics started much earlier than others on the same page, so you don't waste time reading zombie messages or replies to help, question or request topics that were actually asked years ago.",
```

Reply:
Thanks Sam! I support however you think the description should be worded.  Thinking a bit more about the actual implementation, the last phrase should probably be "... actually asked **long** ago" rather than "... actually asked years ago" since the default is ~1 month, not years.

Reply:
Feel free to tweak my description.

Reply:
I'm happy with your description, Sam. "**long** ago" is my only suggestion. Thanks again.